curwmysqladapter/station.py

```diff
@@ -42,6 +43,7 @@ class Station(Enum):
 
         WRF: 100000,
         FLO2D: 100000,
+        MIKE: 1300000,
```

Comment:
@madumalt  This's the range of values that's going to use. So, it should be
```
MIKE: 100000,
```

Reply:
corrected it